src/GitHub.InlineReviews/Tags/ShowInlineCommentGlyph.xaml

```diff
@@ -4,21 +4,38 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              mc:Ignorable="d"
-             ToolTipService.ShowDuration="600000" ToolTipService.HasDropShadow="True">
+             ToolTipService.ShowDuration="600000">
 
-    <UserControl.Resources>
-        <!-- This will be the color of an added/deleted word in the diff view or the indicator margin color -->
-        <SolidColorBrush x:Key="DiffChangeBackground" Color="DarkSlateGray" />
-    </UserControl.Resources>
+    <UserControl.Style>
+        <Style TargetType="UserControl">
+            <Style.Triggers>
+                <DataTrigger Binding="{Binding DiffChangeType}" Value="Add">
+                    <Setter Property="Background" Value="{DynamicResource DiffChangeBackground.Add}" />
```

Comment:
I've created 3 new dynamic brushes that will change according the the theme. They're called `DiffChangeBackground.Add`, `DiffChangeBackground.Delete` and `DiffChangeBackground.None` to mirror the `DiffChangeType` enum.

The old `DiffChangeBackground` is no longer being set for each glyph. Everything is now done via the `DiffChangeType` property.